### ISSUE TITLE
Installation issue: paraview@5.8.1

### ISSUE THREAD

**Issue by jackpap** [LLNL/spack#21198]
### Steps to reproduce the issue

```console
$ spack install paraview@5.8.1%gcc@7.5.0~cuda~examples~hdf5~ipo+kits+mpi+opengl2~osmesa+plugins~python+python3~qt+shared build_type=RelWithDebInfo cuda_arch=native arch=linux-opensuse_leap15-skylake
...
```
### Information on your system

* **Spack:** 0.16.0-860-89b9d07ef
* **Python:** 3.6.12
* **Platform:** linux-opensuse_leap15-skylake
* **Concretizer:** original

### Additional information

VTK/Rendering/OpenGL2/../../../lib64/../lib64/libvtkIO-pv5.8.so.1: undefined reference to `pugi::xml_node::next_sibling() const'

@chuckatkins @danlipsa @vicentebolea 
[spack-build-out.txt](https://github.com/spack/spack/files/5850933/spack-build-out.txt)
[spack-build-env.txt](https://github.com/spack/spack/files/5850934/spack-build-env.txt)
[spack-configure-args.txt](https://github.com/spack/spack/files/5850935/spack-configure-args.txt)

### General information

- [x] I have run `spack debug report` and reported the version of Spack/Python/Platform
- [x] I have run `spack maintainers <name-of-the-package>` and @mentioned any maintainers
- [x] I have uploaded the build log and environment files
- [x] I have searched the issues of this repo and believe this is not a duplicate

I'm new to spack, but this does seem like a real issue. Please let me know if I can assist further.

**Comment by vicentebolea** [comment]
Thanks @jackpap for the ticket, the Paraview team will have a look at this right away.

**Comment by vicentebolea** [comment]
@jackpap this is being resolved in the following PR: https://github.com/spack/spack/pull/21224

**Comment by alalazo** [comment]
Fixed in #21224
